primefaces/src/main/resources/META-INF/resources/primefaces/core/core.widget.js

```diff
@@ -608,6 +608,32 @@ if (!PrimeFaces.widget) {
             }
             
             return this.cfg.formId;
+        },
+
+        /**
+         * Gets a localized label text for this widget.
+         * @param {string} label The label key to look up
+         * @returns {string} The label text - either from the widget configuration if specified,
+         * or from the PrimeFaces global labels
+         */
+        getLocaleLabel: function(label) {
```

Comment:
should we just call it getLabel?

both (non-aria and aria) are locale-labels :D just a different category